### ISSUE TITLE
Demo link does not work

### ISSUE THREAD

**Issue by ioslife** [getBoolean/subete_app#133]
The demo link provided doesn't work, so users cannot see what you're building!

**Comment by getBoolean** [comment]
Yeah I'm aware, web builds are currently broken due to CORS policy.
